src/Drupal/DrupalAutoloader.php

```diff
@@ -112,7 +112,8 @@ public function register(Container $container): void
                 $this->loadAndCatchErrors($module_dir . '/' . $module_name . '.post_update.php');
             }
             // Add .api.php
-            if (file_exists($module_dir . '/' . $module_name . '.api.php')) {
+            $opt_out_of_api_php_scanning = isset($drupalParams['opt_out_api.php']) && (bool) $drupalParams['opt_out_api.php'] === true;
+            if (!$opt_out_of_api_php_scanning && file_exists($module_dir . '/' . $module_name . '.api.php')) {
                 $this->loadAndCatchErrors($module_dir . '/' . $module_name . '.api.php');
```

Comment:
Technically, it will always be set due to schema and defaults. Can `opt_out_of_api_php_scanning` be defined outside of the `foreach` loop.